src/test/java/newbank/TestApp.java

```diff
@@ -127,12 +131,52 @@ public void canHandleUnknownCommands() throws IOException {
     assertThat(response, equalTo("FAIL: Unknown command."));
   }
 
+  private void addCustomer(final String name, final String password) throws IOException {
+    String response = testCommand(String.format("REGISTER %s %s\n", name, password));
+    assertThat(response, containsString("SUCCESS"));
+  }
+
   @Test
   public void canRegisterCustomer() throws IOException {
-    String response = testCommand("REGISTER TestCustomer password1\n");
-    assertThat(response, equalTo("SUCCESS: Customer created successfully."));
+    addCustomer("TestCustomer1", "password1");
+
+    String response = logIn("TestCustomer1", "password1");
+    assertThat(response, containsString("SUCCESS"));
+  }
+
+  private void setupCustomerWithAccount() throws IOException {
+    addCustomer("TestCustomer2", "password2");
+
+    String response = logIn("TestCustomer2", "password2");
+    assertThat(response, containsString("SUCCESS"));
+
+    response = testCommand("NEWACCOUNT Savings\n");
+    assertThat(response, containsString("SUCCESS"));
+  }
+
+  @Test
+  public void canDepositMoney() throws IOException {
```

Comment:
And it would be great to handle the above mentioned failure here.

Reply:
Done.